### ISSUE TITLE
Make --hidecursor the default or give it a short option.

### ISSUE THREAD

**Issue by Earnestly** [naelstrof/maim#52]
This is a suggestion to bring maim more in inline with scrot.  At least
personally and all the people I've spoken to, the default of showing the mouse
pointer only seems to only get in the way of the content (troubleshooting).  It
would be nice if either `--hidecursor` was made the default operation or a more
convenient shortopt was provided.

A workaround for now is to use aliases or wrapper scripts.

**Comment by naelstrof** [comment]
I have to agree with you since even I use --hidecursor by default haha. I put it in the cursor by default due to a special circumstance where I wanted to point out a certain word in a text image with the mouse in a screenshot. I realize the best way for this is to have maim have a delay so that people can move the cursor where they want it, but that's dumb.

You're right, maim should default to hide the cursor. I've already implemented the change on the [experimental branch](https://github.com/naelstrof/maim/tree/experimental). It'll get merged as soon as I figure out how to get the Arch maintainers to not strip all the functionality from community/devil. Or when I just write my own image lib for experimental slop.

You should be able to use the experimental branch for now though.

**Comment by naelstrof** [comment]
OK it's fully implemented with e18b06121a9036ff56c691ab34db7ce8661a5df8 and ab83360b3e88500c00b4eda1b10802769d35529a. Feel free to close this issue if you feel like they're adequate.

**Comment by Earnestly** [comment]
Wonderful, thanks for the quick reponse and even fix o/

**Comment by Earnestly** [comment]
This seems to have been reverted where `-u` is now necessary to explicitly hide the mouse pointer.  Is this intentional?

**Comment by naelstrof** [comment]
Whoops, I accidentally reverted this during my refactor.
I'll fix it in a bit.

**Comment by KeepBotting** [comment]
Hey there! I got here from Google, while I was searching for a way to hide the mouse cursor. It gets in the way when creating screenshots out of very small selections. Adding `--hidecursor` was enough to make the mouse cursor disappear, and solved my issue! Thank you!

**Comment by KeepBotting** [comment]
Cannot reproduce using `maim` 5.5.2, the cursor is hidden as expected when using `--hidecursor`

**Comment by naelstrof** [comment]
Ah jeez, I hate changing stuff like this because people are already probably used to how it works as is, there's no convention saying what's "normal" really.

@KeepBotting I believe they want maim to ignore the cursor by default, not just that the flag is doing what is intended.

I feel real bad that I spaced this (again!), it's something that doesn't bother me at all since maim is kind of meant to be used in other utility scripts.

When I get to it, I'll do what I did last time and add a --showCursor option that overrides the --hideCursor. That way you can alias it to your hearts content, and if you needed to show the cursor you can just specify the flag that shows the cursor. Emulating scrot's functionality perfectly.

However it'll be a while, I'm currently transitioning jobs and doing school still. Again sorry for dropping the ball on this one!

**Comment by Earnestly** [comment]
Any news on this?
